lib/github.sh

```diff
@@ -207,6 +206,23 @@ strap::github::api::token::create() {
 
   # we have a token now - save it to secure storage:
   strap::github::token::save "$username" "$token"
+
+  #Test if we need to do an SSO enable
+  response="$(curl --silent --show-error -i -H "Authorization: token $token" -H 'Content-Type: application/json' https://api.github.com/user/issues)"
+  headers="$(echo "$response" | sed "/^\s*$(printf '\r')*$/q" | sed '/^[[:space:]]*$/d' | tail -n +2)"
+  if echo "$headers" | grep -q 'X-GitHub-SSO:'; then
+    # Linux based system
+    if command -v gio >/dev/null; then
+      gio open https://github.com/settings/tokens
+    # Mac
+    elif command -v open >/dev/null; then
+      open https://github.com/settings/tokens
+    echo "One or more of the Github Orgs you belong to requires that you to enable an SSO personal access token before proceeding. Visit https://github.com/settings/tokens in your web browser and authorize the token"
```

Comment:
This is supposed to be after a closing `fi` statement.  That is, after line 219, there should be a `fi`, and then this block of echos/read.

Reply:
Bah 😢